packages/components/package.json

```
@@ -88,7 +88,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.14 || ^17",
     "react-dom": "^16.14 || ^17",
-    "react-intl": "^5.10.0"
+    "react-intl": "^5.10.0 || ^6.0.0"
```

```suggestion
    "react-intl": "^5.10.0 || ^6"
```

maybe this actually would be better?